pkg/controller/v1beta1/inferenceservice/utils/utils.go

```diff
@@ -186,7 +187,12 @@ func ReplacePlaceholders(container *v1.Container, meta metav1.ObjectMeta) error
 func UpdateImageTag(container *v1.Container, runtimeVersion *string, isvcConfig *v1beta1.InferenceServicesConfig) {
 	image := container.Image
 	if runtimeVersion != nil && len(strings.Split(image, ":")) > 0 {
```

Comment:
I think now the `len(strings.Split(image, ":")) > 0` check is redundant and can be removed.

Reply:
Thanks. I removed it in https://github.com/kserve/kserve/pull/2074/commits/cc1397c3dc15990ef1c757513c42140ba32ac01e